bitnami/elasticsearch/values.yaml

```diff
@@ -130,6 +130,35 @@ discovery:
 #     mountPath: /certs
 #     readOnly: true
 
+## initdb scripts
+## Specify dictionary of scripts to be run at first boot
+## Alternatively, you can put your scripts under the files/docker-entrypoint-initdb.d directory
+##
+# initdbScripts:
+#   my_init_script.sh: |
```

Comment:
Be consistent with the amount of # in the examples. Below you use two but here only one.

Reply:
Changed.